test/performance/latency_test.go

```diff
@@ -95,6 +96,10 @@ func timeToServe(t *testing.T, img, query string, reqTimeout time.Duration) {
 	// Save the json result for benchmarking
 	resp.SaveJSON(tName)
 
+	if len(resp.Result) == 0 {
+		t.Fatalf("No result found for the load test")
+	}
+
 	// Add latency metrics
 	var tc []junit.TestCase
 	for _, p := range resp.Result[0].DurationHistogram.Percentiles {
```

Comment:
do you really need to check for the length above? wouldn't the outofbound error be reported here?

Reply:
this fails with an exception failing the entire workflow. If we check this, only this test should fail and we should be able to run the rest of the tests from the suite

Reply:
I see, then should it be checked everywhere? for example in `scale_revision_by_load_test.go`

Reply:
Yes, probably. But, this is not required for each test. I feel that its a safe assumption to make that if the test ran, it should have some value. Ideally, this should be handled by the library, which I am working on fixing and will update both the tests once that is done.

Reply:
sgtm